platform/darwin/src/MGLConversionValue.h

```diff
@@ -0,0 +1,68 @@
+#include <string>
+
+class MGLConversionValue {
```

Comment:
Good point – we aren't actually using method overloading here, so the fact that both bool and float are encapsulated in NSNumber just means the implementation of isBool() and isNumber() would involve an ||.